test/rabbit.test.ts

```diff
@@ -115,6 +116,26 @@ describe('Test rabbit class', function() {
     stub.calledWith(content, headers, rabbit.channel, `test_${this.name}`, undefined).should.be.true();
   });
 
+  it('should publish to queue by adding prefix_', async function() {
```

Comment:
Is this identical to the previous one? If so just update the name of the previous one.

Reply:
It was a new test based on the idea not to add the prefix if it is the same as the beginning of queue name. 
The idea is dropped so, the test is deleted also. Thanks!